.github/ISSUE_TEMPLATE/documentation_report.md

```diff
@@ -1,61 +1,36 @@
 ---
 name: 📝 Documentation Report
 about: Ask us about docs
-
 ---
-
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
+<!--- Verify first that your improvement is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
 ##### SUMMARY
-<!--- Explain the problem briefly -->
+<!--- Explain the problem briefly below -->
 
 ##### ISSUE TYPE
- - Documentation Report
+- Documentation Report
 
 ##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+<!--- Write the short name of the module, plugin, task or feature below -->
```

Comment:
This could also be the name of an `rst` file.

Reply:
Indeed, there's prior art ;-) Will add.